embassy-rp/src/spi.rs

```diff
@@ -106,15 +106,55 @@ impl<'d, T: Instance, M: Mode> Spi<'d, T, M> {
 
         if let Some(pin) = &clk {
             pin.gpio().ctrl().write(|w| w.set_funcsel(1));
+            pin.pad_ctrl().write(|w| {
+                #[cfg(feature = "rp235x")]
```

Comment:
Looks little suspicious. Is cfg intended here or before whole write? (Same for other three same blocks below ...)

Reply:
This is correct, only the 235x has isolation latches for the gpio pads. The remainder of the pad config is the same.